drizzlepac/haputils/catalog_utils.py

```diff
@@ -1645,15 +1646,16 @@ def identify_sources(self, **pars):
                 log.info("")
                 log.info("The segmentation map contains big sources/islands or a large source fraction of segments.")
                 log.info("Using RickerWavelet2DKernel to generate an alternate segmentation map.")
-                rw2dk = RickerWavelet2DKernel(self.image.kernel_fwhm,
-                                                    x_size=self._rw2d_size,
-                                                    y_size=self._rw2d_size)
-                rw2dk.normalize()
+
+                # Convert the FWHM into a sigma value
+                rw_sigma = self.image.kernel_fwhm * gaussian_fwhm_to_sigma
```

Comment:
```suggestion
                rw_sigma = self.image.kernel_fwhm / gaussian_fwhm_to_sigma
```

Reply:
I believe this is coded correctly "as is".  The **gaussian_fwhm_to_sigma**=0.424660.  In contrast, the **gaussian_sigma_to_fwhm**=2.354820.

Reply:
> I believe this is coded correctly "as is". The **gaussian_fwhm_to_sigma**=0.424660. In contrast, the **gaussian_sigma_to_fwhm**=2.354820.

You are correct!  I was looking at the wrong value -- did not realize there were two constants there.  I will delete both of those change suggestions.